src/bin/bat/config.rs

```diff
@@ -15,6 +16,46 @@ pub fn config_file() -> PathBuf {
         .unwrap_or_else(|| PROJECT_DIRS.config_dir().join("config"))
 }
 
+pub fn generate_config_file() {
```

Comment:
In order to properly handle errors, this function should return a `bat::errors::Result<()>`. In `main.rs`, you would then simply call it with
```rust
generate_config_file()?;
```